test/unit/svc-google-fonts-spec.js

```diff
@@ -43,13 +57,19 @@ describe("Google Font Loader", function() {
       expect(googleFontLoader.getGoogleFonts).to.be.a("function");
     });
 
+    it("should not return fonts that have a subset of 'khmer'", function () {
+      googleFontLoader.getGoogleFonts();
+      $httpBackend.flush();
+
+      expect(googleFontLoader.getGoogleFonts().urls).to.have.lengthOf(1);
+    });
```

Comment:
@stulees It's being explicitly filtered out [here](https://goo.gl/Yo3qSQ) since Google returns it in their API, but it doesn't return the font when requested - https://fonts.googleapis.com/css?family=Buda.